src/librustc_resolve/macros.rs

```diff
@@ -181,6 +181,11 @@ impl<'a> base::Resolver for Resolver<'a> {
     fn find_legacy_attr_invoc(&mut self, attrs: &mut Vec<ast::Attribute>)
                               -> Option<ast::Attribute> {
         for i in 0..attrs.len() {
+            if self.session.plugin_attributes.borrow().iter()
+                    .any(|&(ref attr_nm, _)| attrs[i].check_name(attr_nm)) {
```

Comment:
`check_name` marks the attribute as used when it returns `true`, which is breaking `plugin-attr-register-deny.rs` on Travis.

I would use `attrs[i].name() == attr_nm` instead.